src/registrar/admin.py

```diff
@@ -747,8 +749,59 @@ def organization_type(self, obj):
     search_fields = ["name"]
     search_help_text = "Search by domain name."
     change_form_template = "django/admin/domain_change_form.html"
+    change_list_template = "django/admin/domain_change_list.html"
     readonly_fields = ["state"]
 
+    def export_data_type(self, request):
```

Comment:
Q: Would it be worth leaving a comment on what each CSV is for + how they're sorted? Dunno if that would be useful

ie `domains-by-type - domains in all on hold/dns needed/ready status sorted by domain name`